sponsorship_reporting/__manifest__.py

```diff
@@ -0,0 +1,53 @@
+##############################################################################
+#
+#       ______ Releasing children from poverty      _
+#      / ____/___  ____ ___  ____  ____ ___________(_)___  ____
+#     / /   / __ \/ __ `__ \/ __ \/ __ `/ ___/ ___/ / __ \/ __ \
+#    / /___/ /_/ / / / / / / /_/ / /_/ (__  |__  ) / /_/ / / / /
+#    \____/\____/_/ /_/ /_/ .___/\__,_/____/____/_/\____/_/ /_/
+#                        /_/
+#                            in Jesus' name
+#
+#    Copyright (C) 2023 Compassion CH (http://www.compassion.ch)
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+# pylint: disable=C8101
+{
+    "name": "Compassion Sponsorship Reports",
+    "summary": "Add the reports for the sponsorships",
+    "version": "14.0.1.0.0",
+    "author": "Compassion CH",
+    "category": "Finance",
+    "website": "http://www.compassion.ch",
+    "depends": [
+        "account_fiscal_year",  # OCA/account-financial-reporting
+        "account_fiscal_month",  # OCA/account-financial-reporting
+        "sponsorship_sub_management",  # CompassionCH/compassion-modules
+        "crm_phone",
```

Comment:
Is this really needed? Can we remove it?

Reply:
Yes they are needed, I already reduced the list of dependencies

Reply:
crm_phone? can you let me know why?

Reply:
because of schedule_call method if my memory is good:D

Reply:
I tried without it and it seems to work as good.